crates/templates/src/manager.rs

```diff
@@ -172,7 +172,7 @@ impl TemplateManager {
                     .file_name()
                     .map(|s| s.to_string_lossy().to_string())
                     .unwrap_or_else(|| format!("{}", source_dir.display()));
-                let message = format!("{}", e);
+                let message = format!("{:#}", e);
```

Comment:
Is `{:#}` correct? If so, what does it do? I'm only familiar with it in the context of `{:#?}` (or some alternative spec of printing).

Reply:
It is valid (the alternate display representation, which for errors includes the context stack), but was left in unintentially.  Thanks for flagging.